browser/tor/tor_proxy_config_service.cc

```diff
@@ -27,9 +28,17 @@ const int kTorPasswordLength = 16;
 // Default tor circuit life time is 10 minutes
 constexpr base::TimeDelta kTenMins = base::TimeDelta::FromMinutes(10);
 
+const char kSocksProxy[] = "socks5";
+
+// This is a weird sorta-Windowsism meaning `don't bypass it for the
+// things that bypass it by default'.  For details, see
+// net/proxy_resolution/proxy_bypass_rules.cc.
+const char kTorProxyBypassRules[] = "<-loopback>";
+
 TorProxyConfigService::TorProxyConfigService(
   const std::string& tor_proxy, const std::string& username,
   TorProxyMap* tor_proxy_map) {
+    config_.proxy_rules().bypass_rules.ParseFromString(kTorProxyBypassRules);
```

Comment:
nvm the above comment, I think you can use `AddRulesToSubtractImplicit` to append `<-loopback>` instead of initializing the new rule list

Reply:
Seems to work!